virttest/env_process.py

```diff
@@ -1244,16 +1244,26 @@ def preprocess(test, params, env):
             test.cancel("Got host qemu version:%s, which is not in %s" %
                         (host_qemu, required_qemu))
 
-    # Get the version of bootloader
-    vm_bootloader_ver_cmd = params.get("vm_bootloader_ver_cmd", "")
-    if vm_bootloader_ver_cmd:
-        try:
-            vm_bootloader_ver = a_process.run(
-                vm_bootloader_ver_cmd, shell=True).stdout_text.strip()
-        except a_process.CmdError:
-            vm_bootloader_ver = "Unkown"
-        version_info["vm_bootloader_version"] = str(vm_bootloader_ver)
-        LOG.debug("vm bootloader version: %s", vm_bootloader_ver)
+    # Checking required virtio-win version, if not satisfied, cancel test
+    if params.get("required_virtio_win"):
+        if params.get("cdrom_virtio"):
+            required_virtio_win = params["required_virtio_win"]
+            LOG.info("Checking required virtio-win version: %s" % required_virtio_win)
+            cdrom_virtio = params["cdrom_virtio"]
+            cdrom_virtio_path = os.path.join(data_dir.get_data_dir(), cdrom_virtio)
+            cdrom_virtio_path = utils_misc.get_real_file_path(cdrom_virtio_path)
```

Comment:
Then this should work directly.
```suggestion
            cdrom_virtio_path = os.path.realpath(cdrom_virtio_path)
```

Reply:
Yes,but we need check if it is a link file.If so, then get the realpath, so the `get_real_file_path` still has meaning.

Reply:
Oh yes, we can use `os.path.realpath `directly. If so, we no longer need `get_real_file_path`.